server/rest/license.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from girder.api import access
+from girder.api.describe import Description, describeRoute
+from girder.api.rest import Resource
+
+
+class License(Resource):
+
+    def __init__(self):
+        super(License, self).__init__()
+        self.resourceName = 'license'
+        self.route('GET', (), self.get_licenses)
+
+    @access.public
+    @describeRoute(
+        Description('Return the licenses that a Tale can be under.')
+        .notes('This endpoint returns a list of all of the Whole Tale supported licenses')
+    )
+    def get_licenses(self, params):
+        return [
+            {
+                'name': 'Creative Commons Zero v1.0 Universal',
+                'spdx': 'CC0-1.0',
```

Comment:
This is really the spdx license identifier? You also call it 'license' in the Tale object when it is just a reference to the spdc license identifier.  So there's an inconsistency?   Maybe call it licenseSpdxId or licenseId throughout ?